backend-volto/requirements.txt

```diff
@@ -1,8 +1 @@
 -c constraints.txt
--e src/plone6demo[test]
```

Comment:
How does the package get installed then?

Reply:
It's Cookieplone, you tell me :) but it does get installed.

Reply:
@davisagli, @sneridagh mxdev and mx.ini.